tests/web/Glpi/Inventory/Request.php

```diff
@@ -60,37 +60,46 @@ public function beforeTestMethod($method)
     */
     private function checkResponse(GuzzleHttp\Psr7\Response $res, $reply)
     {
-        $this->integer($res->getStatusCode())->isIdenticalTo(200);
```

Comment:
Why remove this one? It's correct as far as I can see

Reply:
I removed this one as the test should be done by each test as the return code is now different in each cases. The expected value is now tested on lines 83, 102 and 122.